src/main/kotlin/rhmodding/bread/editor/AnimationsTab.kt

```diff
@@ -321,6 +324,37 @@ open class AnimationsTab<F : IDataModel>(editor: Editor<F>) : EditorSubTab<F>(ed
                             }
                         }
                     }
+                    children += Button("Export as PNG sequence").apply {
+                        disableProperty().bind(disableStepControls)
+                        setOnAction {
+                            val ani = currentAnimation
+                            val directoryChooser = DirectoryChooser()
+                            directoryChooser.title = "Export this animation a sequence of PNG images"
+                            directoryChooser.initialDirectory = editor.dataFile.parentFile
+
+                            val dir = directoryChooser.showDialog(null)
+                            if (dir != null) {
+                                var curStep = 0
+                                var file: File
+                                ani.steps.forEach { step ->
```

Comment:
Instead of `forEach` and a separate `curStep` indexing variable, you can use `forEachIndexed` with parameters `curStep, step`. This is good (Kotlin) practice and avoids accidental index variable mismanagement.

```kotlin
ani.steps.forEachIndexed { curStep, step ->
  // ...
}
```